builder/comp-builder.nix

```diff
@@ -119,7 +120,15 @@ let
     [ "--prefix=$out"
       "${haskellLib.componentTarget componentId}"
       "$(cat ${configFiles}/configure-flags)"
-    ] ++ commonConfigureFlags);
+    ] ++ commonConfigureFlags
+      ++ # We don't specify this in 'commonConfigureFlags', as these are also
```

Comment:
Nit: can we put this before `commonConfigureFlags`? At the moment we have `nonCommonFlags ++ commonFlags`, and now we have `nonCommonFlags ++ commonFlags ++ someOtherNonCommonFlags`.